### PR TITLE
mr_checkout: fix usage message

### PR DESCRIPTION
**Pull request by bmeneguele**
This merge request is pretty simple, only adds the '<id>' text to the `lab mr checkout -h` command.
No issue opened for that.

### CODE REVIEW

**Comment by bmeneguele** [comment]
Wait a sec, just noticed issue #364, let me work on that as well.

File: cmd/mr_checkout.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ var (
 
 // listCmd represents the list command
 var checkoutCmd = &cobra.Command{
-	Use:   "checkout",
+	Use:   "checkout <id>",
 	Short: "Checkout an open merge request",
 	Long:  ``,
 	Args:  cobra.ExactArgs(1),
```
Comment (zaquestion) [inline cmd/mr_checkout.go:31]: Now that I can see it infront of me, fixing #364 should just be a matter of:
```suggestion
	Args:  cobra.MinimumNArgs(1),
```

Comment (bmeneguele) [reply cmd/mr_checkout.go:31]: I was thinking in also add the option the handle the "remote" positional argument.
Not sure how desirable it is.. what do you think?

Comment (zaquestion) [reply cmd/mr_checkout.go:31]: Yeah, that's the kinda sad part with this command, it seems to already have support for it, it's just the argument constraint that messes it up.

```
	rn, mrID, err := parseArgs(args)
		rn, mrID, err := parseArgs(args)
		if err != nil {
		if err != nil {
			log.Fatal(err)
			log.Fatal(err)
		}
		}
		mrs, err := lab.MRList(rn, gitlab.ListProjectMergeRequestsOptions{
		mrs, err := lab.MRList(rn, gitlab.ListProjectMergeRequestsOptions{
			IIDs: []int{int(mrID)},
			IIDs: []int{int(mrID)},
		}, 1)
```

**Comment by bmeneguele** [comment]
One thing that I noticed is the checkout misbehaving with the [remote] option when set.
I'm trying to figure out what is actually going on or what I've done wrong. Tomorrow I may have something on that (even though just the reproducer).

**Comment by bmeneguele** [comment]
> One thing that I noticed is the checkout misbehaving with the [remote] option when set.
> I'm trying to figure out what is actually going on or what I've done wrong. Tomorrow I may have something on that (even though just the reproducer).

Will open an issue specific for that..

**Pull request merged**